src/components/IPScan/Summary/serializers.js

```diff
@@ -121,26 +155,28 @@ export const mergeData = (matches, sequenceLength) => {
       })),
       score: match.score,
       residues: undefined,
+      signature: undefined,
     };
-    const residues = match.locations
-      .map(({ sites }) =>
-        sites
-          ? {
-              accession: match.signature.accession,
-              locations: sites.map((site) => ({
-                description: site.description,
-                fragments: site.siteLocations,
-              })),
-            }
-          : null,
-      )
-      .filter(Boolean);
-    if (residues.length > 0) processedMatch.residues = residues;
```

Comment:
A doubt ^^ what did that line do exactly? Why this is not necessary anymore after changes?

Reply:
The changes were calculated in the variable `residues`, however at the end what we return is the variable `processedMatch` so the line is just putting the calculation in the output variable when it found any residues.
This, in the new code, has been changed to line 165, where the residues are now included together with `processedMatch` to get the effect that the residues are part of the entry

Reply:
ok!! thanks 🤗